druid/src/widget/label.rs

```diff
@@ -96,6 +96,9 @@ pub struct Label<T> {
 pub struct RawLabel<T> {
     layout: TextLayout<T>,
     line_break_mode: LineBreaking,
+
+    control_text: bool,
```

Comment:
I would be more explicit about this; it's really only for widget implementors, so I would give it a longer, more explicit name. "is_control_component"?